AirCasting/SearchAndFollow/CompleteScreen/Views/CompleteScreenViewModel.swift

```diff
@@ -44,41 +81,42 @@ class CompleteScreenViewModel: ObservableObject {
     let sessionEndTime: Date
     let sensorType: String
     @Published var sessionStreams: Loadable<[SessionStreamViewModel]> = .loading
-    @Published var chartViewModel = SearchAndFollowChartViewModel(stream: nil)
+    @Published var chartViewModel = SearchAndFollowChartViewModel()
     
-    private let session: SearchSessionResult
-    @Injected private var service: SearchSessionStreamsDownstream
+    private let session: PartialExternalSession
+    private var service = DefaultStreamDownloader()
+    private var isPresented: Binding<Bool>
```

Comment:
I'm not a fan of using bindings for dismissal and much rather prefer the `exitRoute` strategy that we use in other parts of the code